File: solutions/3246-check-if-bitwise-or-has-trailing-zeros/solution.py
class Solution(object):
    def hasTrailingZeros(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        count=0;
        for i in nums:
            if((i&1)==0):
                count+=1;
            if(count>=2):
                return True;
        return False;

File: solutions/3246-check-if-bitwise-or-has-trailing-zeros/solution.c
bool hasTrailingZeros(int* nums, int numsSize) {
    int count=0;
    for(int i=0;i<numsSize;i++)
    {
        if((nums[i]&1)==0)
            count++;
        if(count>=2)
            return true;
    }
    return false;
}
